integration/querier_test.go

```diff
@@ -590,7 +590,7 @@ func testMetadataQueriesWithBlocksStorage(
 				for _, val := range lvt.resp {
 					exp = append(exp, model.LabelValue(val))
 				}
-				require.ElementsMatch(t, exp, labelsRes)
+				require.Equal(t, exp, labelsRes)
```

Comment:
We were using `ElementsMatch()` to avoid flakyness if results are not sorted as the expected value. Is it different now?

Reply:
Yes its different now. We return sorted label values now.